solutions/hierarchy/admin-experimentation/folder.yaml

```diff
@@ -15,9 +15,9 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: Folder
 metadata:
-  name: common-services-infrastructure
+  name: tests.admins.admin1 # kpt-set: tests.admins.${admin-name}
   namespace: hierarchy
 spec:
-  displayName: Common Services Infrastructure
+  displayName: admin1 # kpt-set: ${admin-displayname}
```

Comment:
since everything is now lowercase, the kpt-set can be:

`# kpt-set: ${admin-name}`

Reply:
done